sdk/frontdoor/Azure.ResourceManager.FrontDoor/samples/Generated/Samples/Sample_FrontDoorExperimentResource.cs

```diff
@@ -144,7 +144,7 @@ public async Task GetLatencyScorecardsReport_GetsALatencyScorecardForAGivenExper
         public async Task GetTimeSeriesReport_GetsATimeseriesForAGivenExperiment()
         {
             // Generated from example definition: specification/frontdoor/resource-manager/Microsoft.Network/stable/2019-11-01/examples/NetworkExperimentGetTimeseries.json
-            // this example is just showing the usage of "Reports_GetTimeSeries" operation, for the dependent resources, they will have to be created separately.
+            // this example is just showing the usage of "Reports_GetTimeseries" operation, for the dependent resources, they will have to be created separately.
```

Comment:
it's `series`: https://github.com/Azure/azure-rest-api-specs/blob/49456eb149e3bb5d0528cbca714cfa1806415484/specification/frontdoor/resource-manager/Microsoft.Network/stable/2019-11-01/networkexperiment.json#L762